contracts/pcv/curve/StableSwapOperatorV1.sol

```diff
@@ -190,6 +190,13 @@ contract StableSwapOperatorV1 is PCVDeposit {
         uint256 _lpTotalSupply = IERC20(pool).totalSupply();
         uint256 _lpToWithdraw = _lpTotalSupply * _amount3crvToWithdraw / IStableSwap(pool).balances(_3crvIndex);
 
+        // overshoot withdraw by the Curve withdraw fee, so that the discrepancy
+        // between amountUnderlying and actual amount out is minimized, when we
+        // will call withdraw_liquidity_one_coin() in the 3pool.
+        // the fee is 0.015% (see SwapTemplateMeta.vy#L916) :
+        // fee: uint256 = self.fee * N_COINS / (4 * (N_COINS - 1))
+        _lpToWithdraw = _lpToWithdraw * 100015 / 100000;
```

Comment:
What if the _lpToWithdraw is equal to the total before applying this transformation? It should be capped at the total balance of the pcv deposit.

Reply:
I initially added a check on that, but it has a side effect that I don't like : if the deposit has 1000 DAI, and we try to withdraw 2000, the call does not revert, it just withdraws 1000. What do you think : should we revert, or withdraw the max available ?